src/index.html

```diff
@@ -6,9 +6,108 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Kosugi&family=Roboto:wght@300;500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="#home"
+        class="header__logo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="logo"
+          >
+      </a>
+
+      <div class="menu__mobile hide">
+        <img
+          src="./images/menu_icon.png"
+          alt="menu icon"
+          class="menu__mobile--icon"
+        >
+      </div>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="list__item">
+            <a
+              class="nav__link is-active link--noMargin"
+              href="#apple"
+            >
+            apple
```

Comment:
Don't forget to check the checklist.
Similar formatting should be corrected throughout the file.
![image](https://github.com/mate-academy/layout_moyo-header/assets/54860641/559366c3-c237-42c7-949c-8260455e7663)

Reply:
+